src/migrations/20200228144719-product.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
```

Comment:
Let's remove the semicolons here (make sure to run the linter).

Reply:
Fixed